core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -198,6 +198,15 @@ public interface NativeConfig {
      */
     Optional<Boolean> containerBuild();
 
+    /**
+     * Explicit configuration option to generate Position Independent Executable (PIE) native executables for Linux.
+     * If the system supports, the default behaviour is to generate native binaries without PIE for
```

Comment:
If the system supports what?

I suggest:
```suggestion
     * If the system we are building on supports PIE generation, the default behaviour is to disable it 
```